releaselinks.go

```diff
@@ -99,8 +101,10 @@ func (s *ReleaseLinksService) GetReleaseLink(pid interface{}, tagName string, li
 //
 // GitLab API docs: https://docs.gitlab.com/ee/api/releases/links.html#create-a-link
 type CreateReleaseLinkOptions struct {
-	Name *string `url:"name" json:"name"`
-	URL  *string `url:"url" json:"url"`
+	Name     *string `url:"name" json:"name"`
+	URL      *string `url:"url" json:"url"`
```

Comment:
Not your change, but let's fix it while were add it 😉 
```suggestion
	Name     *string `url:"name,omitempty" json:"name,omitempty"`
	URL      *string `url:"url,omitempty" json:"url,omitempty"`
```